test/sys/test_ptrace.rs

```diff
@@ -1,9 +1,14 @@
+#![cfg(all(target_os = "linux", any(target_arch = "x86",
```

Comment:
This is already specified in `test/sys/mod.rs`, why are you respecifying it here? You probably want to restrict just your new tests a little more rather than the whole module.

Reply:
Ok, I don't remember. Fixed that.